packages/core/src/contracts/tcr/voting.ts

```diff
@@ -175,6 +176,22 @@ export class Voting extends BaseWrapper<CivilPLCRVotingContract> {
     return false;
   }
 
+  public async getRevealedVoteEvent(
+    pollID: BigNumber,
+    voter: EthAddress,
+  ): Promise<
+    DecodedLogEntryEvent<CivilPLCRVoting.Args._VoteRevealed, CivilPLCRVoting.Events._VoteRevealed> | undefined
+  > {
+    if (await this.didRevealVote(voter, pollID)) {
+      const reveal = await this.instance
+        ._VoteRevealedStream({ pollID }, { fromBlock: 0 })
```

Comment:
No filtering by the voter, it's gonna return any kind of reveal instead of first reveal by that specific voter
``` { pollID, voter }, ...```

Reply:
dang good catch